content/implementation-path.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Implementation Path"
+---
+
+The work to be done in support of naming changes is generally low intensity, but can have consequences. We suggest the following:
```

Comment:
I don't exactly understand what "low intensity" means in this context. Can it be rephrased?

Reply:
Low effort, not a huge investment of engineering time.. in that realm. I'll try another phrase.